### DISCUSSION TITLE
Custom widget implements ReactWidget，how to save by ctrl + s？

### DISCUSSION THREAD

**Discussion by hyy215** [eclipse-theia/theia#9385] [Q&A] [answered]
Custom widget implements ReactWidget，how to save by ctrl + s？
implements SourceSavable?

**Comment by vince-fugnitto** [comment, marked as answer]
Marking the discussion as answered since it is a duplicate of:
- https://community.theia-ide.org/t/custom-widget-implements-reactwidget-how-to-save-the-file/1635.

**Reply by hyy215** [reply]
Thank you for your help! The problem is resolved.
